src/main/java/seedu/duke/backend/Ui.java

```diff
@@ -42,9 +46,9 @@ public void run() {
             // Placeholder if additional routine is required when user enters incorrect parameters
         } catch (DukeNoMatchException ne) {
             printOutput("No such command. Try 'help' for a list of commands.");
-        } catch (Exception e) {
+        } /*catch (Exception e) {
             printOutput("Command execution failed with an unhandled error!", true);
-        }
+        }*/
```

Comment:
Not sure why this is commented out

Reply:
Sorry, I commented out for testing some exceptions. Forgot to recover it.